src/patternfly/components/Table/table.scss

```diff
@@ -4,6 +4,7 @@
 // TODO: update grouping comments to // * Table {element}
 
 @include pf-root($table) {
+  --#{$table}--LineHeight: calc(var(--pf-t--global--font--line-height--body) * var(--pf-t--global--font--size--body--default));
```

Comment:
Since this is applied to cells, any text in a cell that 1) is not `font--size--body--default` and 2) inherits `line-height` will have the wrong line-height. Our empty state demo shows this - https://patternfly-pr-7244.surge.sh/components/table/html-demos/empty-state/

`empty-state__body` font size is 16px and the line-height is 21px now (14px * 1.5) instead of 24px.